docs/src/introduction/sway_quickstart.md

```diff
@@ -195,6 +200,8 @@ TransactionId(HexFormatted(7cef24ea33513733ab78c5daa5328d622d4b38187d0f0d1857b27
 
 Note the contract ID — you will need it if you want to build out a frontend to interact with this contract.
 
+If `forc-deploy` return `no such subcommand` error, try to install it manually <a href="https://github.com/FuelLabs/forc-client/releases" target="_blank"> here </a>
+
```

Comment:
Ok I see, we can leave this part for those who want to install with `cargo`, and add another section using `fuelup` what do you think? It can be done today

Reply:
The standalone repository for `forc-client` has since been deprecated - `forc-client` is actually downloaded alongside the core `forc`. I believe this error was a result of that period of time where we were trying to extract `forc-client` out of the Sway repository, and it wasn't packaged together by default in those specific versions, but this should no longer occur now.

 The correct link to the release page should be https://github.com/FuelLabs/sway/releases, but even so I think we should suggest (in order of preference) using: fuelup > cargo > manual installation.

Reply:
Definitely, order of priority should be `fuelup`, `cargo` then manual installation
But in order to have a complete documentation that accompanies the developers in any configuration, having the three scenarios will be useful as well. 
I'm gonna work on it in the next few days.